turbinia/workers/__init__.py

```diff
@@ -99,6 +148,16 @@ def add_evidence(self, evidence):
     """
     self.evidence.append(evidence)
 
+  def save_local_file(self, file_):
+    """Saves local file by writing to all non-local output writers.
+
+    Args:
+      file_ (string): Path to file to save.
+    """
+    for writer in self._output_writers:
+      if writer.name != 'LocalOutputWriter':
```

Comment:
No;  Since the file already exists locally, we don't need to use the localOutputWriter to write it again.